src/init/population.rs

```diff
@@ -294,21 +318,23 @@ impl TuPerson {
                 salary_hourly: self.salary_hourly,
                 salary_yearly: self.salary_yearly,
             },
-
-            bmi: match self.bmi.as_str() {
```

Comment:
Previously, the `Person` struct in Rust had each of these health fields defined. They're also defined in the protobuf, and in `protobuf.rs`, we were doing a very straightforward transformation. But there's no need for the distinction; we can just directly use the protobuf-generated code and fill that out. That's what this change is doing.